brayns/common/camera/Camera.h

```diff
@@ -68,39 +64,26 @@ class Camera : public PropertyObject
     {
         _updateValue(_position, position);
     }
-
     /**
        Gets camera position
        @return The x, y, z coordinates of the camera position
     */
     const Vector3d& getPosition() const { return _position; }
     /**
-       Sets camera target
-       @param target The x, y, z coordinates of the camera target: the point the
-              camera is "looking at" or focused on
-    */
-    void setTarget(const Vector3d& target) { _updateValue(_target, target); }
-    /**
-       Gets camera target
-       @return The x, y, z coordinates of the camera target: the point the
-               camera is "looking at" or focused on
-    */
-    const Vector3d& getTarget() const { return _target; }
-    /**
-       Sets camera up vector
-       @param up the x, y, z coordinates of the up vector's end point
-    */
-    void setUp(const Vector3d& up) { _updateValue(_up, up); }
-    /**
-       Gets camera up vector
-       @return the x, y, z coordinates of the up vector's end point
+       Sets camera orientaion quaternion.
+       @param quat The orientation quaternion
     */
-    const Vector3d& getUp() const { return _up; }
+    void setQuaternion(const Quaterniond& quat)
+    {
+        Quaterniond normalized = quat;
```

Comment:
could just take a copy arg in the function then

Reply:
Done